scan/src/components/modal/submitTx/SendMsg.re

```diff
@@ -70,6 +70,7 @@ let make = (~address, ~receiver, ~setMsgsOpt) => {
       msg="Recipient Address"
       code=true
       placeholder="Insert recipient address"
+      autoFocus=true
```

Comment:
If the modal is opened with the recipient address, then focus on the amount's input.
If not, then focus on the recipient address input.